### ISSUE TITLE
[BUG] Not able to select a custom snooze time

### ISSUE THREAD

**Issue by pscofield** [rohanb10/snoozz-tab-snoozing#66]
## Description
When I select the "Choose your own time..." option, I am consistently not able to modify the time at which it wakes up. It defaults to the current time (when the popup opens) but I am not able to change it.

There are options for "±5mins", and "±1hour" but they don't work either. Clicking on them doesn't change anything. Moreover, I am not able to inline-edit the wakeup time. I have attached a screenshot to share what I see.

## Expected Behavior
I should be able to edit the time to any particular time with or without using the ±5mins/±1h buttons.

## Actual Behavior
The wakeup time cannot be changed from the default.

## How to Reproduce (if applicable)
Description above. Screenshot attached below.

## Your Device
* Browser and version: Google Chrome Version 94.0.4606.81 (Official Build) (x86_64)
* OS: MacBook Pro
* Device Resolution (UI related bugs only)

## Which Page
Popup.

## Screenshots / Videos / Logs
![image](https://user-images.githubusercontent.com/4214369/137823688-a37d6662-3af8-420e-8947-89e2a2baa0a3.png)

P.S. big fan and a big user of this extension. Thanks for creating it! <3

**Comment by pscofield** [comment]
Hey. Sorry I got late getting back to this. I have shared a screen record of my experience here = https://imgur.com/a/eqzVtQg. This screen record is for the Chrome Version `95.0.4638.69 (Official Build)(x86_64)` on m`acOS Big Sur v11.6`.

In the video, you'll notice the issue that I am trying to explain. The screen record doesn't capture my click actions but whenever my mouse hovers goes to one of the buttons I was clicking it.

I also noticed that the ±1 day buttons work reliably but the mins hours buttons don't. Also, the "Choose" your own time GUI works fine for me when I go to the ["nap room page"](chrome-extension://lklendgldejcnkkaldoggoapclkepgfb/html/nap-room.html) -> 3-dots button -> "More actions" -> Edit Snooze Time -> Choose your own time.

Thanks in advance for resolving this issue. I already love the extension.
